devops/Makefile

```diff
@@ -231,31 +231,31 @@ build-db-migrations:
 	test -n "$(BUILD_REF)"
 	test -n "$(DB_MIGRATIONS_BUILD_REF)"
 	@echo "+\n++ BUILDING DB migrations with tag: $(BUILD_REF)\n+"
-	@oc -n $(BUILD_NAMESPACE) process -f $(BUILD_TEMPLATE_PATH) -p TAG=$(BUILD_REF) -p SOURCE_REPOSITORY_REF=$(BUILD_REF) -p BASE_IMAGE_NAME="nodejs-16" -p BASE_IMAGE_TAG="1" -p BASE_IMAGE_REPO="artifacts.developer.gov.bc.ca/redhat-access-docker-remote/ubi8/" -p SOURCE_CONTEXT_DIR=$(SOURCE_CONTEXT_DIR)backend -p DOCKER_FILE_PATH=apps/db-migrations/Dockerfile -p NAME=$(DB_MIGRATIONS_BUILD_REF) | oc -n $(BUILD_NAMESPACE) apply -f -
+	@oc -n $(BUILD_NAMESPACE) process -f $(BUILD_TEMPLATE_PATH) -p TAG=$(BUILD_REF) -p SOURCE_REPOSITORY_REF=$(BUILD_REF) -p BASE_IMAGE_NAME="nodejs-18" -p BASE_IMAGE_TAG="1-71.1697652955" -p BASE_IMAGE_REPO="artifacts.developer.gov.bc.ca/redhat-access-docker-remote/ubi8/" -p SOURCE_CONTEXT_DIR=$(SOURCE_CONTEXT_DIR)backend -p DOCKER_FILE_PATH=apps/db-migrations/Dockerfile -p NAME=$(DB_MIGRATIONS_BUILD_REF) | oc -n $(BUILD_NAMESPACE) apply -f -
```

Comment:
Why can't we use tag `1` like before? they point to same version

![image](https://github.com/bcgov/SIMS/assets/54600590/664f83dd-e008-49d5-b2ce-e2befad19167)

Reply:
This tag is recommended by RedHat:
![image](https://github.com/bcgov/SIMS/assets/78114138/87960f4e-9999-4adf-956b-85de8a76abbd)

Version 1-15 (from a year ago): 
![image](https://github.com/bcgov/SIMS/assets/78114138/c2e0e967-2f42-4c88-98bc-b011ca4d1140)
Compared to 1-71.1697652955 (latest): 
![image](https://github.com/bcgov/SIMS/assets/78114138/0d5ed80a-3c14-4bd0-b206-af324f025fc3)

1-71 tag has 12 important security adivisories:
![image](https://github.com/bcgov/SIMS/assets/78114138/8a038877-e1c3-45a3-bf56-d319b2fe2f19)

Reply:
Isn't `1-71.1697652955` | `latest` | `1` the same? I was trying to say we can use  `1` instead of the long version `1-71.1697652955` and also I believe `1` must hold latest stable version of the node-18 image.

Reply:
Holding the latest stable version would be good but that must happen manually and it must be under our control, that is the same reason that we do not use `latest` all around 😉

Reply:
OK, after our talk I got what you mean. Well, ,we can use tag `1` and that would get the latest version for major version 1. 
Before we were using version [1-5](https://catalog.redhat.com/software/containers/rhel8/nodejs-16/615aee893f6014fa45ae1ae1?image=6183969c3d73db3b14b37e7c&architecture=amd64) and that one doesn't point to any latest version. 
@andrewsignori-aot @guru-aot @ann-aot @cditcher What do you guys suggest? Use `1` to always get the latest version or have a fixed tagged version?